Makefile

```
@@ -10,9 +10,10 @@ build:
 	sudo podman push $(IMAGE)
 
 deploy:
-	sed "s#__IMAGE__#${IMAGE}#g;s#__BM_INVENTORY_URL__#${BM_INVENTORY_URL}#g" deploy/ocp-metal-ui-template.yaml > deploy/ocp-metal-ui.yaml
-	kubectl apply -f deploy/ocp-metal-ui.yaml
-	rm deploy/ocp-metal-ui.yaml
+	mkdir -p build/deploy/
+	tools/deploy_config.sh "${BM_INVENTORY_URL}" "${IMAGE}" deploy/ocp-metal-ui-template.yaml > build/deploy/ocp-metal-ui.yaml
```

Shouldn't the output go to `deploy/ocp-metal-ui.yaml` rather than `build/deploy/...`?
```suggestion
	tools/deploy_config.sh "${BM_INVENTORY_URL}" "${IMAGE}" deploy/ocp-metal-ui-template.yaml > deploy/ocp-metal-ui.yaml
```

---

The build directory should contain only built static files which are then served from the nginx server. We should not include these files there.

---

It is usually good practice to only modify files under the build/ dir in the build process.